### PR TITLE
added the AJAX Session Expiration Handling page

### PR DESCRIPTION
**Pull request by kennyj42**
We just released the ability to add an additional header to AJAX requests in order to receive a 401. Adding this as companion documentation.

AUTH-3149

### CODE REVIEW

**Review by jroyal — APPROVED** [review]

File: products/access/src/content/advanced-management/AJAX-Session-Expiration.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+order: 3
+---
+
+# AJAX Session Expiration Handling
```
Comment (adamschwartz) [inline products/access/src/content/advanced-management/AJAX-Session-Expiration.md:5]: Casing: `# AJAX session expiration handling`

File: products/access/src/content/advanced-management/AJAX-Session-Expiration.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+order: 3
```
Comment (adamschwartz) [inline products/access/src/content/advanced-management/AJAX-Session-Expiration.md:2]: Add `title: AJAX session expiration` for abbreviated side nav title.

File: products/access/src/content/advanced-management/AJAX-Session-Expiration.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+# AJAX Session Expiration Handling
+
+Pages that rely heavily on AJAX or single page applications can block sub-requests due to an expired Access token without prompting the user to re-authenticate.
+
+You can configure Access to provide a 401 response on sub-requests with an expired session token. We recommend using this response code to either force a page refresh or display a message to the user that their session has expired.
+
+In order to receive a 401 for an expired session, add the following header to all AJAX requests:
+
+`X-Requested-With: XMLHttpRequest`
```
Comment (adamschwartz) [inline products/access/src/content/advanced-management/AJAX-Session-Expiration.md:13]: Consider changing to:

``````
```txt
X-Requested-With: XMLHttpRequest
```
``````

File: products/access/src/content/advanced-management/AJAX-Session-Expiration.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+order: 3
+---
+
+# AJAX Session Expiration Handling
+
+Pages that rely heavily on AJAX or single page applications can block sub-requests due to an expired Access token without prompting the user to re-authenticate.
+
+You can configure Access to provide a 401 response on sub-requests with an expired session token. We recommend using this response code to either force a page refresh or display a message to the user that their session has expired.
+
+In order to receive a 401 for an expired session, add the following header to all AJAX requests:
```
Comment (adamschwartz) [inline products/access/src/content/advanced-management/AJAX-Session-Expiration.md:11]: Consider referring to 401 as <code>\`401\`</code>.

**Review by adamschwartz — CHANGES_REQUESTED** [review]
Nits.

**Comment by adamschwartz** [comment]
Please note: As part of the work for https://github.com/cloudflare/cloudflare-docs/issues/156 I updated this PR to be based off of the `staging` branch instead of the `master` branch.

**Comment by adamschwartz** [comment]
Not sure what happened here. This PR will need to be made again against the new `production` branch. My apologies.

**Pull request closed without merging**